Directory.Build.props

```
@@ -50,6 +50,11 @@
     <DisableImplicitComponentsAnalyzers>true</DisableImplicitComponentsAnalyzers>
   </PropertyGroup>
 
+  <!-- Enable implicit usings -->
+  <PropertyGroup>
+    <ImplicitUsings>enable</ImplicitUsings>
```

One maybe thing to do would be to turn it off if there's more than one TFM. It'll avoid fighting VS tooling. 

```suggestion
    <ImplicitUsings Condition="''$(TargetFrameworks)' == ''">enable</ImplicitUsings>
```
I guess that would require specifying this in the Directory.Build.targets